src/components/Close.js

```diff
@@ -19,6 +19,7 @@ function Close({ className, onClick }) {
 Close.propTypes = {
   className: PropTypes.string.isRequired,
   onClick: PropTypes.func.isRequired,
+  ariaLabel: PropTypes.string,
```

Comment:
Should we have a default value for this?

Reply:
what do you think, based on your audits and experience?

Reply:
I think it makes sense to have a default of `aria-label="Close"`. The only thing is that it wouldn't be localized/internationalized/translated and would be in English. However, it can be easily configured. I would add it but I don't feel that strongly about it.

Reply:
@elrumordelaluz Added `defaultProp` for `closeButtonAriaLabel`. If it looks good, can you please merge? Thanks!